ui-kit/src/constants/colors.ts

```diff
@@ -0,0 +1,28 @@
+export const colors = {
+  green50: '#13BC4C',
+  green40: '#78CF81',
+  green60: '#0F933C',
+  blue50: '#135CE9',
+  blue40: '#87ADF5',
+  blue60: '#013CAD',
+  red50: '#CB121C',
+  red40: '#F29CA1',
+  red60: '#9B0B13',
+  yellow50: '#F0CB08',
+  yellow40: '#F9E681',
+  yellow60: '#AA8F00',
+  gray100: '#1B1B1C',
+  gray90: '#2A2A2C',
+  gray80: '#4C4D53',
+  gray70: '#76777D',
+  gray60: '#8E9095',
+  gray50: '#B5B6B9',
+  gray40: '#D0D1D3',
+  gray30: '#E3E4E5',
+  gray20: '#F3F4F5',
+  gray10: '#FCFCFD',
+} as const;
+
+export type SemanticColorName = 'positive' | 'informative' | 'negative' | 'notice';
+
+export type ColorProperty = keyof typeof colors;
```

Comment:
이 타이핑 대로라면
```ts
const props: Props = {
  foo: 'green40',
  bar: 'blue40',
}
```
처럼 써야하는데 혹시 의도하신 건가요? 이러면 사용자가 `green40`이나 `blue40`이라는 값이 있다는 걸 알기 전에는 사용하기 어려울 것 같습니다. 첫 글자를 쳐야 자동완성이 되니까유

그때 보내주셨던 예시대로 요렇게 변경하는건 어떨까요?

```suggestion
export type ColorProperty = typeof colors[keyof typeof colors];
```
```ts
interface Props {
  foo: ColorProperty;
  bar: ColorProperty;
}

const props: Props = {
  foo: '#013CAD',
  bar: colors.blue40,
}
```

이렇게 타이핑하면 `colors` 맵의 존재만 알고 있다면 내부에 어떤 색이 있는지도 IDE에서 자동으로 트래킹해서 자동완성으로 보여줍니다...!

Reply:
아 이 타입은 스토리파일에서 저렇게 프로퍼티 리스트만 빼온 타입이 필요해서 작성했어요
언급해주신 방법대로의 타입 정의 추가하는거를 침대에 눕고나서 생각나서,
아 오늘 아침에 추가해야지 하다가 깜빡했네요 😅